clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -431,56 +431,46 @@ raw2trace_t::merge_and_process_thread_files()
                     if (entry.timestamp.type != OFFLINE_TYPE_TIMESTAMP)
                         FATAL_ERROR("Missing timestamp entry");
                     times[i] = entry.timestamp.usec;
-                    VPRINT(3, "Thread %u timestamp is @0x" ZHEX64_FORMAT_STRING
-                           "\n", (uint)tids[i], times[i]);
+                    VPRINT(3, "File %u timestamp is @0x" ZHEX64_FORMAT_STRING
+                           "\n", i, times[i]);
                 }
                 if (times[i] != 0 && times[i] < min_time) {
                     min_time = times[i];
                     next_tidx = i;
                 }
             }
-            VPRINT(2, "Next thread in timestamp order is %u @0x" ZHEX64_FORMAT_STRING
-                   "\n", (uint)tids[next_tidx], times[next_tidx]);
```

Comment:
Why was this removed?  I found it useful for seeing the operation: which file is next.